### PR TITLE
Adds pagination params

### PR DESCRIPTION
**Pull request by rjz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,15 @@ body of the header is a JSON object containing pagination information:
...
 
+Subsequent pages from the collection endpoint may be retrieved by setting the
+`page` query parameter:
+
+```bash
+curl https://stack.versal.com/api2/courses/123/userdata?page=2 \
+  -H 'SID: 28438e94-480d-11e3-95fd-ce3f5508acd9'
+```
+
+
```
Comment (pacharya) [inline README.md]: there is a public api test in https://github.com/Versal/rest-api/blob/master/src/test/resources/csv/tests/vfo/public-course-api-via-access-key.csv ; this new method to be added to it?

Comment (rjz) [inline README.md]: A little confused—tests already exist for `/userdata`, and the pagination parameters are well-covered elsewhere.

Comment (pacharya) [inline README.md]: Are we sure there is no complex interaction between the pagination and the authentication?

Comment (pacharya) [inline README.md]: er this is a public repo. removing this comment. please do teh same, RJ

**Pull request merged**